crypto/benches/transaction.rs

```diff
@@ -0,0 +1,74 @@
+use criterion::async_executor::FuturesExecutor;
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
+use openmls_traits::types::Ciphersuite;
+use rand::distributions::{Alphanumeric, DistString};
+
+pub mod utils;
+
+use utils::*;
+fn decrypt_transaction(c: &mut Criterion) {
+    const MESSAGE_LENGTH: usize = 100;
+    const MESSAGE_COUNT: usize = 1000;
+
+    let mut group = c.benchmark_group(format!(
+        "Decrypting {MESSAGE_COUNT} messages, transaction vs. no transaction"
+    ));
```

Comment:
This group name shows up in the output, and it's not obvious which case is running for any given results. Recommend creating the group slightly later within the `transactional` loop, and naming it like `Decrypting {MESSAGE_COUT} messages: {id}`.

Reply:
@coriolinus the output looks like this:
`Decrypting 1000 messages, transaction vs. no transaction/transactional` and
`Decrypting 1000 messages, transaction vs. no transaction/non-transactional`, where the first is transactional sending and the second non-transactional.
I'd like to keep the group creation in that place, as I follow the pattern of the other existing benches. I admint those might need an overhaul, though.

However, I made the output a bit more concise and added the information of message length in bytes.

Reply:
Yes, that's clearer now, thanks!